src/helpers/generatePosts.ts

```diff
@@ -0,0 +1,28 @@
+import { User } from '../types/User';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+
+import postsFromServer from '../api/posts';
+import commentsFromServer from '../api/comments';
+import usersFromServer from '../api/users';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+function getCommentsById(postId: number): Comment[] | null {
```

Comment:
you can left empty array
```suggestion
function getCommentsById(postId: number): Comment[] {
```